tmt/steps/provision/__init__.py

```diff
@@ -200,7 +200,7 @@ def details(self) -> None:
         if self.opt('dry'):
             return
 
-        self.info('full name', self.full_name, 'green')
+        self.info('full name', self.multihost_name, 'green')
```

Comment:
@happz should we also not call it `multihost name` to the outside world?

Reply:
Absolutely, fixed in ffd520e3